run-basic-batch-operators-test.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/sh
+
+# Test run timestamp
+TIMESTAMP=$(date | sed 's/[ :]/_/g')
+
+# base folder
+BASE_DIR=/var/www/html
+
+# INPUTS
+
+# config.yaml template file path
+CONFIG_YAML_TEMPLATE=$(pwd)/tnf_config.yml.template
+
+# Docker config used to pull operator images
+DOCKER_CONFIG=config.json
+
+# Location of telco/non-telco classification file
+CNF_TYPE=cmd/tnf/claim/show/csv/cnf-type.json
+
+# operator bundle list from user
+ORIG_BUNDLE_PATH=$1
```

Comment:
It should be assigned after checking if the number of parameters is correct.

Reply:
fixed